src/pages/(study)/schedule/Manage/components/MemberInfoSection.tsx

```diff
@@ -0,0 +1,84 @@
+import { Check, X } from 'lucide-react';
+import { useAttendanceStudyQuery } from '../hooks/useAttendanceStudyQuery';
+import { ErrorBoundary } from '@/components';
+import { UserAvatar } from '@/components/user';
+
+type MemberInfoSectionProps = {
+  study_id: number;
+};
+
+const MemberInfoSection = ({ study_id }: MemberInfoSectionProps) => {
+  const { data: members } = useAttendanceStudyQuery({ study_id });
+  const scheduleDates =
+    members.members[0]?.attendance.map((a) => a.schedule_date) ?? [];
```

Comment:
변수명이랑 key명이 동일해서(`members`) 가독성이 좋지 않게 느껴집니다.

코드를 보니 members.members 값만 참조를 하고 있네요.
```suggestion
  const { data } = useAttendanceStudyQuery({ study_id });
  const { members } = data;
  const scheduleDates =
    members[0]?.attendance.map((a) => a.schedule_date) ?? [];
```

위와 같이 필요한 값만 추출해서 사용하시면 가독성이 더 좋아질 것 같아요.